projects/ngx-d3timeline/src/lib/ngx-d3timeline.service.ts

```diff
@@ -79,4 +96,18 @@ export class NgxD3TimelineService implements OnDestroy {
   private zoomed() {
     this.store.dispatch(new fromActions.ZoomedAction(event));
   }
+
+  setupResizing() {
```

Comment:
This method should be above `private zoomed()`

Reply:
done